cartoframes/data/observatory/repository/country_repo.py

```diff
@@ -0,0 +1,36 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_country_repo():
+    return CountryRepository()
+
+
+class CountryRepository(object):
+
+    def __init__(self):
+        self.client = RepoClient()
+
+    def get_all(self):
+        return self._to_countries(self.client.get_countries())
+
+    def get_by_id(self, iso_code3):
+        result = self.client.get_countries('country_iso_code3', iso_code3)
+
+        if len(result) == 0:
+            return None
+
+        return self._to_country(result[0])
+
+    @staticmethod
+    def _to_country(result):
+        from cartoframes.data.observatory.country import Country
```

Comment:
I prefer doing the `import` on the top of the file. Are you trying to avoid `the circular pain`?

Reply:
Yes, see my comment above. This is the only way I found to make it work without changing the API. I think it's a design flaw due to mix model and behavior :(